webapp/src/ts/services/telemetry.service.ts

```diff
@@ -119,9 +119,10 @@ export class TelemetryService {
     return Promise
       .all([
         this.dbService.get().get('_design/medic-client'),
-        this.dbService.get().query('medic-client/doc_by_type', { key: ['form'], include_docs: true })
+        this.dbService.get().query('medic-client/doc_by_type', { key: ['form'], include_docs: true }),
+        this.dbService.get().get('settings')
```

Comment:
The settings document is very large, and you don't actually use the whole contents here. 
Could this be changed so you request `allDocs` with the settings doc id as the key and just take the rev from there? 

```suggestion
        this.dbService.get().allDocs({ key: 'settings' })
```

and then 

```
settings: allDocsResult?.rows?.[0].value?.rev,
```

Reply:
Thanks for the suggestion @dianabarsan. I have made the change.